### PR TITLE
add streamlit-aggrid

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timkpaine** [comment]
supersedes https://github.com/conda-forge/staged-recipes/pull/18646

**Comment by timkpaine** [comment]
@conda-forge/help-python all set

File: recipes/streamlit-aggrid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.7,<4.0
+  run:
+    - pandas >=1.2
```
Comment (dopplershift) [inline recipes/streamlit-aggrid/meta.yaml:23]: Upstream lists:
```ini
    install_requires=[
        "streamlit >= 0.75",
        "simplejson >= 3.0",
        "pandas",
        "numpy"
```
Is there any reason this recipe doesn't match?

Comment (timkpaine) [reply recipes/streamlit-aggrid/meta.yaml:23]: im using the pyproject requirements
https://github.com/PablocFonseca/streamlit-aggrid/blob/main/pyproject.toml#L14

Comment (dopplershift) [reply recipes/streamlit-aggrid/meta.yaml:23]: Alright then. 🤷‍♂️

**Review by dopplershift — APPROVED** [review]

**Comment by dopplershift** [comment]
@maxibor Can you confirm that you're willing to be listed as a maintainer for this recipe?

**Comment by timkpaine** [comment]
> @maxibor Can you confirm that you're willing to be listed as a maintainer for this recipe?

he did a PR separately so i think its ok

**Comment by maxibor** [comment]
LGTM 🙂

**Pull request merged**